src/lib/Polykey.ts

```diff
@@ -1,232 +1,39 @@
 import os from 'os'
-import fs from 'fs'
-import Path from 'path'
-import crypto from 'crypto'
-import jsonfile from 'jsonfile'
-import Vault from '@polykey/Vault'
-import KeyManager from '@polykey/KeyManager'
-import PeerStore from '@polykey/PeerStore/PeerStore'
-import PeerInfo from '@polykey/PeerStore/PeerInfo'
-import RPCMessage from '@polykey/RPC/RPCMessage'
-import PeerDiscovery from '@polykey/P2P/PeerDiscovery'
-
-type Metadata = {
-  vaults: {
-    [vaultName: string]: {
-      key: Buffer, tags: Array<string>
-    }
-  }
-  peerInfo: Uint8Array
-  publicKeyPath?: string
-  privateKeyPath?: string
-}
+import GitServer from '@polykey/git/GitServer'
+import KeyManager from '@polykey/keys/KeyManager'
+import PeerManager from '@polykey/peers/PeerManager'
+import PeerInfo, { Address } from '@polykey/peers/PeerInfo'
+import VaultManager from '@polykey/vaults/VaultManager'
 
 class Polykey {
   polykeyPath: string
-  private fs: typeof fs
-  private vaults:Map<string, Vault>
-  private metadata: Metadata
-  private metadataPath: string
 
+  vaultManager: VaultManager
```

Comment:
I deliberately removed the private scoping to match what was there for the `KeyManager` class as well as make it easy for users to access the different managers. I have had no issues with losing scope because there are no methods in the `PolyKey` class that need binding.